### PR TITLE
Added login page

### PR DESCRIPTION
**Pull request by e-azocar**
### Description

I created the login view with 2 inputs and 1 button.

**ONLY DESKTOP VIEW**

#### Screenshots
image

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

1. Go to '/login'

### Checklist:

- [x] Doesn't break the current code.
- [x] Passes linters and test, also is building.
- [x] Doesn't have spelling or grammatical problems.
- [x] Doesn't have unnecessary comments or debugging code.
- [x] The branch is updated with the last dev branch changes.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -33,7 +33,9 @@
         "bracketSpacing": true,
         "arrowParens": "avoid"
       }
-    ]
+    ],
+    "import/no-unresolved": ["off"],
```
Comment (Israel-Laguan) [inline .eslintrc.json:37]: Why this is necesary?

File: .eslintrc.json [hunk 2]
```diff
@@ -33,7 +33,9 @@
...
+    "@next/next/no-img-element": "off"
```
Comment (Israel-Laguan) [inline .eslintrc.json:38]: Hmm I get the problems with it, but why not use the image from next?

File: tsconfig.json [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-  "exclude": ["node_modules"]
-}
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": false,
```
Comment (Israel-Laguan) [inline tsconfig.json:8]: Hmm can you explain this one please?

File: .eslintrc.json (hunk 1 above)
Comment (e-azocar) [reply .eslintrc.json]: ## Without this line Eslint show me this error
image

File: tsconfig.json (hunk 3 above)
Comment (e-azocar) [reply tsconfig.json:8]: Without this line Eslint show me this error:
image

File: .eslintrc.json (hunk 1 above)
Comment (Israel-Laguan) [reply .eslintrc.json]: Hmm is it because that path doesn't exists? SHould be GraphQL << note the uppercase L

File: tsconfig.json (hunk 3 above)
Comment (Israel-Laguan) [reply tsconfig.json:8]: I think the problem is the folder GraphQl, please check that

File: .eslintrc.json (hunk 1 above)
Comment (e-azocar) [reply .eslintrc.json]: I didn't note it, Thanks

**Comment by Israel-Laguan** [comment]
Closed in favor of #7

**Pull request closed without merging**